src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,228 @@
+
+
+        // when
+        // then
+        assertThat(createLine(lineRequest).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+    }
+
+    /**
+     * Given: 여러 개의 지하철 노선이 등록되어 있고,
+     * When: 관리자가 지하철 노선 목록을 조회하면,
+     * Then: 모든 지하철 노선 목록이 반환된다.
+     */
+    @DisplayName("지하철 노선 목록을 조회한다.")
+    @Test
+    void showLines() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        assertThat(createLine(lineRequestRed).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        LineRequest lineRequestGreen = new LineRequest();
+        lineRequestGreen.setName("분당선");
+        lineRequestGreen.setColor("bg-green-600");
+        lineRequestGreen.setUpStationId(1);
+        lineRequestGreen.setDownStationId(3);
+        lineRequestGreen.setDistance(20);
+
+        assertThat(createLine(lineRequestGreen).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        // when
+        List<String> lineNames =
+                given().log().all()
+                        .when().get("/lines")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
+        //then
+        assertThat(lineNames).contains("신분당선", "분당선");
+    }
+
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 조회하면,
+     * Then: 해당 노선의 정보가 반환된다.
+     */
+    @DisplayName("지하철 노선을 조회한다.")
+    @Test
+    void showLine() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        assertThat(createLine(lineRequestRed).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        // when
+        List<String> lineNames =
+                given().log().all()
+                        .when().get("/lines")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
+        //then
+        assertThat(lineNames).contains("신분당선");
+    }
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 수정하면,
+     * Then: 해당 노선의 정보가 수정된다.
+     */
+    @DisplayName("지하철 노선을 수정한다.")
+    @Test
+    void editLine() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        String lineId  =   createLine(lineRequestRed).getHeaders().get("Location").toString().split("/")[2];
```

Comment:
```suggestion
        String lineId  =   createLine(lineRequestRed).getHeader("Location");
```

위와 같이 수정하면 lineId 가 아닌 `/lines/{id}` 자체가 추출됩니다 😅

Reply:
id를 추출해서 넣을 생각만 하고 url 자체를 추출해서 넣을 생각을 못했네요 😅 
수정했습니다!